optimum/modeling_base.py

```diff
@@ -395,7 +394,7 @@ def from_pretrained(
         elif export and trust_remote_code is None:
             trust_remote_code = False
 
-        from_pretrained_method = cls._from_transformers if export else cls._from_pretrained
+        from_pretrained_method = cls._export if export else cls._from_pretrained
```

Comment:
This assumes that `_export` has been implemented

Reply:
yeah, ifwe merge this, we might need to do a sync release for optimum and all sub-packages. Everyone changes.

Reply:
okay sounds good. so at this rate we could leave the `@abstractmethod`?

Reply:
also the case for optimum-furiosa https://github.com/huggingface/optimum-furiosa/blob/bc77f8408cc5b7ceb66044bd6bc30c38fbf2c568/optimum/furiosa/modeling_base.py#L195 cc @mht-sharma